lib/Extension/LanguageServer/Handler/DebugHandler.php

```diff
@@ -137,6 +142,7 @@ public function status(): Promise
             '  documents: ' . $this->workspace->count(),
             '  services: ' . (string)json_encode($this->serviceManager->runningServices()),
             '  diagnostics: ' . (string)$this->diagnosticProvider->name(),
+            '  watcher: ' . $this->watcher->describe(),
```

Comment:
the only "useful" thing about the welcome message was showing the watcher status, but I'm not sure if anybody really cared much about it. I've added it to the status report instead.